pkgs/development/libraries/gperftools/default.nix

```diff
@@ -30,27 +31,58 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook ];
 
   # tcmalloc uses libunwind in a way that works correctly only on non-ARM dynamically linked linux
-  buildInputs = [ perl ]
-             ++ lib.optional (stdenv.isLinux && !(stdenv.hostPlatform.isAarch || stdenv.hostPlatform.isStatic )) libunwind;
+  buildInputs =
+    [ perl ]
+    ++ lib.optional (
+      stdenv.isLinux && !(stdenv.hostPlatform.isAarch || stdenv.hostPlatform.isStatic)
+    ) libunwind;
 
   # Disable general dynamic TLS on AArch to support dlopen()'ing the library:
   # https://bugzilla.redhat.com/show_bug.cgi?id=1483558
-  configureFlags = lib.optional stdenv.hostPlatform.isAarch
-    "--disable-general-dynamic-tls";
+  configureFlags = lib.optional stdenv.hostPlatform.isAarch "--disable-general-dynamic-tls";
 
   prePatch = lib.optionalString stdenv.isDarwin ''
     substituteInPlace Makefile.am --replace stdc++ c++
   '';
 
-  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin
-    "-D_XOPEN_SOURCE";
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-D_XOPEN_SOURCE";
 
   # some packages want to link to the static tcmalloc_minimal
   # to drop the runtime dependency on gperftools
   dontDisableStatic = true;
 
   enableParallelBuilding = true;
 
+  # split output to minimize dependencies on dynamic linking.
+  # tcmalloc_minimal goes into a dedicated output to further minimize
+  # the runtime dependencies of applications that just use tcmalloc
+  # as an allocator, without the profiling.
+  outputs = [
```

Comment:
outputs are usually placed near the beginning of the package definition, like after version

Reply:
Thanks. I moved the outputs to the top, after version.